poetry.toml

```diff
@@ -0,0 +1,3 @@
+[virtualenvs]
+create = true
+in-project = true
```

Comment:
This configuration might not be suitable for all developers - to my knowledge, I believe that the `poetry config` command can store it's global config options, such as these, outside of the project itself. Are you able to remove this please?